src/steamship/agents/llm/openai.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from steamship import Block, PluginInstance, Steamship
+from steamship.agents.base import LLM
+
+
+class OpenAI(LLM):
+    generator: PluginInstance
+    client: Steamship
+
+    def __init__(self, client, *args, **kwargs):
```

Comment:
We can leave this as a follow-on, but I think it's hugely valuable to permit plumbing in the model name so that folks can select 3.5-turbo if they want